### DISCUSSION TITLE
What do we call a baby stingray?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#842] [Galaxy Trivia] [answered]
**Bonus:** What name is given to a young female 🦃?

**Comment by peterbaccaro** [comment, marked as answer]
A baby stingray is commonly referred to as a "**pup**" or a "**stingray pup**". The term "pup" is often used to describe the offspring of various marine animals, including stingrays.

A young female turkey is called a "**jenny**" or a "**jennet**". These terms are used to specifically refer to juvenile or young female turkeys.

**Comment by SANKARAMDAS** [comment]
Although docile creatures, they inflict more human injuries yearly than any other animal in Amazonian rivers. Individuals in a litter of stingrays are called pups.
A female turkey is called a hen, if the female is in her first year she is called a Jenny. Juvenile females are called jennies.
